tests/functional/pyocf/types/cache.py

```diff
@@ -383,6 +383,7 @@ def get_stats(self):
                 "core_count": cache_info.core_count,
                 "metadata_footprint": Size(cache_info.metadata_footprint),
                 "metadata_end_offset": Size(cache_info.metadata_end_offset),
+                "cache_id": self.owner.lib.ocf_cache_get_id(self),
```

Comment:
At this point cache is unlocked - there's a possibility that it's already gone.

Reply:
Done